docs/intro/install.rst

```diff
@@ -34,6 +32,19 @@ To install using pip::
 Platform specific installation notes
 ====================================
 
+Anaconda
+--------
+
+.. note::
+
+  For Windows and OS X users, this is the recommended way to install Scrapy.
```

Comment:
+1 to recommend in case pip doesn't work.

2015-10-07 13:58 GMT-03:00 Julia Medina <notifications@github.com>:

> In docs/intro/install.rst
> <https://github.com/scrapy/scrapy/pull/1516#discussion_r41416046>:
>
> > @@ -34,6 +32,19 @@ To install using pip::
> >  Platform specific installation notes
> >  ====================================
> >
> > +Anaconda
> > +--------
> > +
> > +.. note::
> > +
> > +  For Windows and OS X users, this is the recommended way to install Scrapy.
>
> I agree that it's probably better to stick with pip for OS X. Maybe we
> could recommend this method to anyone who had problems install Scrapy with
> pip, regardless of the platform (and say it explicitly in this note)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scrapy/scrapy/pull/1516/files#r41416046>.
>